docker/integration_tests/configs/scripts/auth/clientScriptAuth.zst

```diff
@@ -0,0 +1,65 @@
+{
+  "about": "This is a Zest script. For more details about Zest visit https://github.com/zaproxy/zest/",
+  "zestVersion": "0.8",
+  "title": "RecordedTestAuth",
+  "description": "",
+  "prefix": "",
+  "type": "StandAlone",
+  "parameters": {
+    "tokenStart": "{{",
+    "tokenEnd": "}}",
+    "tokens": {},
+    "elementType": "ZestVariables"
+  },
+  "statements": [
```

Comment:
Would be good to have custom requests as well to make sure those are correctly handled. The user can use any script with custom behaviour/logic. I'm good if this is done in other PR.

Reply:
I'm not sure what you're looking for but yes let's leave it for another PR.

Reply:
Zest scripts can have statements to send requests, which will not happen in the browser.